docs/001_develop/03_client-capabilities/005_grids/003_grid-pro/index.mdx

```diff
@@ -36,35 +37,6 @@ For more detailed information on API and configurations, please refer to the [AP
 - **State persistence**: ability to save and restore grid state.
 - **Row models**: support for both client-side and server-side [row models](https://www.ag-grid.com/javascript-data-grid/row-models/).
 
-## Getting Started
```

Comment:
question:

are we doing this in all other places?

Feels odd having this bits at the end, not everyone will start from seed/create, it's worth mentioning those but not sure about moving to the bottom of this page

Reply:
Yeah we're trying to tell the story of each component at first and as most people will have it pre-installed we're leading with usage and examples. Plus once someone has installed it they don't need to see it as the first section every time they open the docs

Long term we can change it back through if we get feedback around it